src/app_functions_sdk_py/utils/environment.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2024-2025 IOTech Ltd
+# SPDX-License-Identifier: Apache-2.0
+
+""""
+This module provides utility functions for handling environment variables.
+"""
+
+import os
+
+from app_functions_sdk_py.contracts.clients.logger import Logger
+
+ENV_EDGEX_MSG_BASE64_PAYLOAD = "EDGEX_MSG_BASE64_PAYLOAD"
```

Comment:
Rename the constant to `ENV_KEY_MSG_BASE64_PAYLOAD` and move this constants along with other ENV_KEY constants as defined in src/app_functions_sdk_py/bootstrap/environment.py to src/app_functions_sdk_py/constants.py to avoid cyclic import.

Reply:
moved ENV_X constants to src/app_functions_sdk_py/constants.py